xc7/utils/prjxray_form_channels.py

```diff
@@ -207,38 +203,36 @@ def import_grid(db, grid, conn):
         # tile: pkey name tile_type_pkey grid_x grid_y
         c.execute(
             """
-INSERT INTO tile(name, tile_type_pkey, grid_x, grid_y)
+INSERT INTO phy_tile(name, tile_type_pkey, grid_x, grid_y)
 VALUES
   (?, ?, ?, ?)""",
             (tile, tile_types[gridinfo.tile_type], loc.grid_x, loc.grid_y)
         )
 
-    build_other_indicies(c)
```

Comment:
Aren't we missing the index on phy_tile(name) and phy_tile(grid_x, grid_y)?

Reply:
I will add them. But they are barely needed as the phy_tile table is accessed only once inside the `remap_tile_grid` function.